### PR TITLE
run hack/update-vendor.sh to fix verify v1.27

### PR DESCRIPTION
**Pull request by pacoxu**
xref master #119083

```release-note
None
```

### CODE REVIEW

**Comment by jiahuif** [comment]
/assign @dims
I think you have the context. Could you take a look at this too?
/triage accepted

File: staging/src/k8s.io/api/go.mod [hunk 1]
```diff
@@ -21,7 +21,7 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/rogpeppe/go-internal v1.10.0 // indirect
+	github.com/rogpeppe/go-internal v1.11.0 // indirect
```
Comment (liggitt) [inline staging/src/k8s.io/api/go.mod:24]: didn't this already merge in https://github.com/kubernetes/kubernetes/pull/119086/commits/39a4cd1a083f0dff5ff3fc78063b64ad3c61099f

Comment (pacoxu) [reply staging/src/k8s.io/api/go.mod:24]: Yes. After #119086 is merged, this PR can be closed.
/close

**Pull request closed without merging**